apps/files_sharing/src/mixins/SharesMixin.js

```diff
@@ -135,6 +135,15 @@ export default {
 		isShareOwner() {
 			return this.share && this.share.owner === getCurrentUser().uid
 		},
+		isExpiryDateEnforced() {
+			if (this.isPublicShare) {
+				return this.config.isDefaultExpireDateEnforced
+			}
+			if (this.isRemoteShare) {
+			    return this.config.isDefaultRemoteExpireDateEnforced
+			}
+			return this.config.isDefaultInternalExpireDateEnforced
+		},
```

Comment:
Does `isDefaultExpireDateEnforced` only apply to public shares? If not, then this should be better: 
```suggestion
		isExpiryDateEnforced() {
			if (this.isPublicShare) {
				return this.config.isDefaultExpireDateEnforced
			}
			if (this.isRemoteShare) {
			    return this.config.isDefaultRemoteExpireDateEnforced || this.config.isDefaultExpireDateEnforced
			}
			return this.config.isDefaultInternalExpireDateEnforced || this.config.isDefaultExpireDateEnforced
		},
```